### DISCUSSION TITLE
Bimi record support for domains

### DISCUSSION THREAD

**Discussion by parishahla** [postalserver/postal#3128] [Development discussions]
Hello there.  As an ESP, Postal has been the backbone of our business. Is it possible to add bimi record to each domain? If this feature isn't available yet, what are the best ways to provide it to our users?
Thanks!

**Comment by willpower232** [comment]
I can't say I've heard of it before but it sounds like you just link to an image in your DNS so presumably you could host an image on your website and link to that? Not sure what help Postal would be in this situation.

https://support.google.com/a/answer/10911321?hl=en
